Tests/SwiftDocCTests/Test Bundles/DictionaryData.docc/dictionary.symbols.json

```diff
@@ -239,6 +239,32 @@
     },
     {
       "accessLevel": "public",
+      "allowedValues": [
+        "Classic Rock",
+        "Folk",
+        null
```

Comment:
Question: Is this decoded as a null value in the JSON or as a string? I see that the test expects it as a string. How is this expected to be displayed on the page?

Reply:
JSON allows proper null/nil values and that is what is being indicated here. Internally it stays flagged as `null` up until the render JSON phase where it is converted to a string for display. There isn't an obvious way to flag that in the rendered page unless we force all strings to be quoted, which isn't ideal. It is up to the reader to recognize that and entry that reads `null` means the nil value and not the string "null".